example/matrixMulWithMdspan/src/matrixMulMdSpan.cpp

```diff
@@ -0,0 +1,178 @@
+/* Copyright 2023 Mehmet Yusufoglu, Bernhard Manfred Gruber, René Widera, Andrea Bocci, Simeon Ehrig
```

Comment:
Needs to be modiefied

Reply:
ok changed to 2024 and SPDX-License-Identifier: MPL-2.0